angr/procedures/libc/gets.py

```diff
@@ -0,0 +1,34 @@
+import angr
```

Comment:
It seems that this is the only file that is relevant to this PR (`SimProcedure of gets`). Can you please put all other changes in separate PRs, and only leaving this file in this PR?